src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1019,12 +1019,17 @@ def load_arguments(self, _):
         c.argument('permissions', arg_type=get_enum_type(GallerySharingPermissionTypes), arg_group='Sharing Profile',
                    min_api='2020-09-30', is_experimental=True,
                    help='This property allows you to specify the permission of sharing gallery.')
+        c.argument('soft_delete', arg_type=get_three_state_flag(), min_api='2021-03-01', is_experimental=True,
+                   help='Enables soft-deletion for resources in this gallery, '
+                        'allowing them to be recovered within retention time.')
```

Comment:
Please use the first person voice in the help message
```suggestion
                   help='Enable soft-deletion for resources in this gallery, '
                        'allowing them to be recovered within retention time.')
```